test/unit/airtap.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape')
+const path = require('path')
+const exec = require('child_process').exec
+const airtap = path.resolve(__dirname, '../../bin/airtap.js')
+const messages = require('../../lib/messages')
+
+test('exits cleanly and does nothing if no secure travis env', t => {
+  const env = Object.assign({}, process.env, {
+    TRAVIS_SECURE_ENV_VARS: 'false'
+  })
+  exec(airtap, { env }, (err, stdout, stderr) => {
```

Comment:
To support Windows I think this should be `exec('node ${airtap}', ..)`. Opening an issue.